src/main/java/com/commercetools/sync/products/utils/ProductUpdateActionUtils.java

```diff
@@ -912,21 +910,10 @@ private static boolean isSameForAllActionNew(
     final UpdateAction<Product> addVariant =
         AddVariant.of(draft.getAttributes(), draft.getPrices(), draft.getSku(), true)
             .withKey(draft.getKey())
-            .withImages(draft.getImages());
+            .withImages(draft.getImages())
+            .withAssetDrafts(draft.getAssets());
 
     actions.add(addVariant);
```

Comment:
Do you still need this actions list?

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/714/commits/a737dbf424865929542715f3c2d76cda4d0502b8